packages/graph/deployment/postgres/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM postgres:14-alpine
+
+COPY init-user-db.sh /docker-entrypoint-initdb.d/init-user-db.sh 
```

Comment:
I tried to mount this script as a volume in the docker-compose file, to avoid the need for a custom `Dockerfile` - however this gave me a permissioning error which I couldn't figure out.